src/utils.ts

```diff
@@ -53,7 +53,18 @@ export const CHROM_SIZE_HG19 = {
     chrM: 16571
 };
 
-function getChromInterval(chromSize: { [k: string]: number }) {
+export const getAbsoluteMutationPosition = (
+    assembly: string,
+    chromosome: string,
+    relativeMutationPosition: number
+): number => {
+    const chrIntervals = getChromInterval(assembly === 'hg19' ? CHROM_SIZE_HG19 : CHROM_SIZE_HG38); // assembly is defined in the Data Config of Chromoscope.
+    const [start, end] = chrIntervals[chromosome]; // start is the offset of chr10 in the linearlized genomic coordinates
+
+    return start + relativeMutationPosition; // information from the clinical data config
+};
```

Comment:
While we are using this for point mutations at the moment, the function is generic enough that it can be used in other data. I would suggest renaming the function and making the order of parameters consistent with the `getRelativeGenomicPosition`.

```suggestion
export const getAbsoluteGenomicPosition = (
    chromosome: string,
    relativePosition: number,
    assembly: 'hg38' | 'hg19' // ← only two options supported
): number => {
    const chrIntervals = getChromInterval(assembly === 'hg19' ? CHROM_SIZE_HG19 : CHROM_SIZE_HG38); // assembly is defined in the Data Config of Chromoscope.
    const [start, end] = chrIntervals[chromosome]; // start is the offset of chr10 in the linearlized genomic coordinates

    return start + relativePosition; // information from the clinical data config
};
```